### DISCUSSION TITLE
How to get sales order being late to be started as early as possible

### DISCUSSION THREAD

**Discussion by tjheiska** [frePPLe/frepple#493] [Q&A]
Hello,

In attached example data I have two sales orders that are both going to be late. The one with higher priority is started as early as possible, but the second one is not. Any idea what am I doing wrong with modeling as there should be plenty of resources left?

[frepple-20220211055538.log](https://github.com/frePPLe/frepple/files/8046248/frepple-20220211055538.log)
[frepple (3).xlsx](https://github.com/frePPLe/frepple/files/8046244/frepple.3.xlsx))

**Comment by jdetaeye** [comment]
I believe this is a bug.  Issue https://github.com/frePPLe/frepple/issues/494 is created.
